app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -555,6 +557,10 @@ class BrowserTabViewModel @Inject constructor(
     val title: String?
         get() = site?.title
 
+    val isSplitOmnibarEnabled: Boolean by lazy {
+        omnibarFeatureRepository.isSplitOmnibarAvailable && settingsDataStore.omnibarType == OmnibarType.SPLIT
```

Comment:
nit: Couldn't this whole check be encapsulated in the `omnibarFeatureRepository`?